assets/ui/scoreHud.ui

```diff
@@ -7,7 +7,7 @@
             {
                 "type": "UIBox",
                 "layoutInfo": {
-                    "width": 300,
+                    "width": 400,
                     "height": 100,
                     "position-horizontal-center": {},
```

Comment:
When replacing the lines about `width` and height` with
```JSON
"use-content-width": true,
"use-content-height": true,
```
the whole HUD element moves to the left, despite the `"position-horizontal-center": {},`, which is kind of confusing 🤔

Reply:
My first guess is that this probably has to do with the outside `relativeLayout` not having `layoutInfo`, but I haven't looked too much into it yet.